.github/workflows/api-rds-plan.yml

```diff
@@ -7,6 +7,13 @@ on:
   workflow_dispatch: # Allow manual trigger
 
 jobs:
+  check-terraform-fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./actions/setup-tfenv-terraform
+      - run: terraform fmt -check -diff -recursive terraform/services/api-rds
+
   terraform-plan:
     needs: check-terraform-fmt
     permissions:
```

Comment:
AB2D also includes a "prod-test" environment with a database. The matrix below will need to be extended with the environment.